common/module.go

```diff
@@ -36,3 +36,33 @@ func (a Module) Path() string {
 
 	return fmt.Sprintf("%s/%s", a.Label, a.Version)
 }
+
+// ModuleObjectNameToFieldName is a grouping of ObjectName to response field name mappings defined for each Module.
+//
+// Explanation: modules have objects, each object is located under certain field name in the response body.
+// This mapping stores the general relationship between the said ObjectName and FieldName.
+// Those objects that do not follow the pattern described in the fallback method are hard code as exceptions.
+//
+// Ex:
+//
+//	Given:	Connector has 2 modules -- Commerce, Messaging.
+//			Commerce module has objects stored under "data" field name, except "carts".
+//			Messaging module has objects stored under the same name as object, except "chats".
+//	Then:	It will be represented as follows:
+//
+//		ModuleObjectNameToFieldName{
+//			ModuleCommerce: handy.NewDefaultMap(map[string]string{
+//				"carts": "carts",
+//			},
+//				func(objectName string) string {
+//					return "data" // always under "data" field {"data": [{},{},...]}
+//				},
+//			),
+//			ModuleHelpCenter: handy.NewDefaultMap(map[string]string{
+//				"chats":        "active_chats",
+//			}, func(objectName string) string {
+//				fieldName := objectName // Object "messages" is stored under {"messages": [{},{},...]}
+//				return fieldName
+//			}),
+//		}
+type ModuleObjectNameToFieldName map[ModuleID]handy.DefaultMap[string, string]
```

Comment:
@Rajat you have asked about the explanation in different comment thread.

First of all I moved this type into `common`. I see that it will be reused by those connectors with multiple modules and inconsitent `ObjectName` to `FieldName` relationship.